src/box/lua/config/init.lua

```diff
@@ -80,13 +80,18 @@ function methods._alert(self, alert)
     table.insert(self._alerts, alert)
 end
 
+function methods._meta(self, source, key, value)
```

Comment:
Nit: If it is a source name, let's name it `source_name`. We use `source` as a source object just below.

Reply:
Thanks, fixed!